dicom_standard/parse_lib.py

```diff
@@ -30,8 +30,7 @@
 ID_PATTERN = re.compile(r'\b(' + '|'.join(NONSTANDARD_SECTION_IDS) + r').+\b')
 SHORT_DICOM_URL_PREFIX = "http://dicom.nema.org/medical/dicom/current/output/chtml/part03/"
 
-MetadataTableType = Dict[str, Any]
-JsonDataType = Union[List[MetadataTableType], MetadataTableType]
+GenericTableType = Union[List[Dict[str, Any]], Dict[str, Any]]
```

Comment:
:+1: I think this renaming makes a lot of sense and conveys what the type represents in the application.